frontend/src/metabase/components/form/FormWidget.jsx

```diff
@@ -41,6 +42,7 @@ const WIDGETS = {
   hidden: FormHiddenWidget,
   textFile: FormTextFileWidget,
   model: FormModelWidget,
+  category: CategoryFieldPicker,
```

Comment:
IMO, a few different UI components can act as category input: for instance, a dropdown or radio buttons. Should we maybe give this component a name that's a bit more focused on the visual appearance of the widget?